tools/appveyor.psm1

```diff
@@ -347,11 +347,8 @@ function Invoke-AppVeyorTest
         Write-Host -Foreground Green 'Running all CoreCLR tests..'
     }
 
-    # Remove telemetry semaphore file in CI
-    $telemetrySemaphoreFilepath = Join-Path $env:CoreOutput DELETE_ME_TO_DISABLE_CONSOLEHOST_TELEMETRY
-    if ( Test-Path "${telemetrySemaphoreFilepath}" ) {
-        Remove-Item -Force ${telemetrySemaphoreFilepath}
-    }
+    #  Disable telemetry in CI
+    $env:POWERSHELL_TELEMETRY_OPTOUT = 1
```

Comment:
Setting 1 to the env variable means you enable the telemetry. You should set 0 here.